examples/angular/tsconfig.json

```diff
@@ -15,8 +15,7 @@
     "lib": ["es2018", "dom"],
     "esModuleInterop": true,
     "paths": {
-      "@aws-amplify/ui-angular": ["../../packages/angular/dist/ui-angular/*"],
-      "@aws-amplify/ui-core": ["../../packages/core/*"],
+      "@aws-amplify/ui-angular": ["../../packages/angular/dist/ui-angular"],
```

Comment:
Due to how angular outputs file to this subdirectory, this has to be explicitly stated through paths (else it'd go to the source files).

Reply:
Have you tried setting the package.json `main` property to the dist folder output file? 

IIRC if you set your `package.json` to the correct `main` property, then the `@aws-amplify/ui-angular` import will default to that path not the src folder (even apps in the yarn workspace). So you wouldn't have too explicitly set this in the `tsconfig` file.

Reply:
Yeah, just tried it now! You were right, that correctly routes the imports to `dist`. But then it was having trouble locating "@aws-amplify/ui-angular/theme.css", because it's not part of the `main` file. 

This is frustrating because `dist/ui-angular` has different `package.json` from `lib/ui-angular`, with its own "main", "module", etc entries so workspaces aren't working as intended. If they were in the same directory, redirecting files through workspaces would've been much easier... But I think we need to use `path` so that it finds the correct directory to reference its package.json.

We can discuss this offline as needed.

Reply:
OK!

Reply:
This is how I got `@aws-amplify/ui-react/styles.css` to resolve correctly:

https://github.com/aws-amplify/amplify-ui/blob/47255d081883cd42c8b11c6198c9175139aa8323/packages/react/package.json#L8-L11

Reply:
Ah, thanks! I'll add that to my package.json. 

But from dev side, this still has the same problem of divided subdirectories vs workspaces:

Reference to `@aws-amplify/angular` goes to `packages/angular/projects/ui-angular` as defined by workspaces, but this should really head to `packages/angular/dist/ui-angular` which has the auto-generated `package.json` that defines all the module declarations. This is why I see these structure as anti pattern, it works poorly against workspaces. But community has solved these nuances with `path` and `publishConfig.directory`